web/src/main/java/se/inera/intyg/webcert/web/service/access/DraftAccessServiceImpl.java

```diff
@@ -175,7 +173,8 @@ public AccessResult allowToPrintDraft(AccessEvaluationParameters accessEvaluatio
                 TsBasEntryPoint.MODULE_ID,
                 TsDiabetesEntryPoint.MODULE_ID,
                 Tstrk1009EntryPoint.MODULE_ID,
-                TsTrk1062EntryPoint.MODULE_ID)
+                TsTrk1062EntryPoint.MODULE_ID,
+                DbModuleEntryPoint.MODULE_ID)// You should be able to print a draft for a deceased person
```

Comment:
I added this because the Certificate and MODULE_ID had no obvious connection. A more fitting name would be excludeModuleEntryPointForDeceased.

Reply:
I do agree that EntryPoint doesn't make so much sense, but what the module Id equals the type of the certificate. Anyhow, not sure that your comment adds much value anyway. And because git will track changes any developer can look in the history if you want to see why that line of code is there.